readline-complete.el

```diff
@@ -135,10 +135,18 @@
 ;; editline instead of readline. Consider recompiling bash, python,
 ;; etc with readline support instead.
 
+;; *** Note: emacs contained a bug from ~24.3.91 to ~24.5-rc3-fixed where
+;; completion on longer lines would fail and possibly erase your prompt.
+
 ;; Patches welcome! https://github.com/monsanto/readline-complete.el
 
 ;;; Code:
 
+(when (or (and (= 24 emacs-major-version) (>= 3 emacs-minor-version))
+          (and (= 25 emacs-major-version) (<= 0 emacs-minor-version)))
```

Comment:
hmm after thinking about it a bit more this is probably more trouble than its worth. I personally run Emacs HEAD but others probably don't. I'll leave it up to you